doc/MANUAL.md

```diff
@@ -89,6 +89,7 @@
       - [Support for Version Range (`~=`) in `requirements.txt`](#support-for-version-range--in-requirementstxt)
     - [Perl](#perl)
     - [PHP](#php)
+    - [Dart](#dart)
```

Comment:
Docs update in the manual looks good.  Can you also add dart to the table in README.md to complete the docs?